src/Resources/views/style/httplug.css.twig

```diff
@@ -46,7 +46,7 @@
     border-radius: 2px 0px 0px 2px;
     border: 0;
     line-height: inherit;
-    background-color: #eee;
+    background-color: var(--metric-value-background);
```

Comment:
i am no CSS expert, but if i understand correctly, this would be compatible with Symfony 3.4 if you provide the old value as default:

```suggestion
    background-color: var(--metric-value-background, #eee);
```